### PR TITLE
GLTFLoader minor clean up

### PR DESCRIPTION
**Pull request by takahirox**
Commenting inline

### CODE REVIEW

File: examples/js/loaders/GLTFLoader.js [hunk 1]
```diff
@@ -368,26 +368,26 @@ THREE.GLTFLoader = ( function () {
 	 *
 	 * PR: https://github.com/KhronosGroup/glTF/pull/1163
 	 */
-	function GLTFMaterialsUnlitExtension( json ) {
+	function GLTFMaterialsUnlitExtension() {
```
Comment (takahirox) [inline examples/js/loaders/GLTFLoader.js]: No argument need here.

File: examples/js/loaders/GLTFLoader.js [hunk 2]
```diff
@@ -368,26 +368,26 @@ THREE.GLTFLoader = ( function () {
...
 
 		this.name = EXTENSIONS.KHR_MATERIALS_UNLIT;
 
 	}
 
-	GLTFMaterialsUnlitExtension.prototype.getMaterialType = function ( material ) {
+	GLTFMaterialsUnlitExtension.prototype.getMaterialType = function () {
```
Comment (takahirox) [inline examples/js/loaders/GLTFLoader.js]: No argument need here.

File: examples/js/loaders/GLTFLoader.js [hunk 3]
```diff
@@ -368,26 +368,26 @@ THREE.GLTFLoader = ( function () {
...
 
 		return THREE.MeshBasicMaterial;
 
 	};
 
-	GLTFMaterialsUnlitExtension.prototype.extendParams = function ( materialParams, material, parser ) {
+	GLTFMaterialsUnlitExtension.prototype.extendParams = function ( materialParams, materialDef, parser ) {
```
Comment (takahirox) [inline examples/js/loaders/GLTFLoader.js]: Consistent naming may be better.

File: examples/js/loaders/GLTFLoader.js [hunk 4]
```diff
@@ -563,7 +563,7 @@ THREE.GLTFLoader = ( function () {
 	 *
 	 * Specification:
 	 */
-	function GLTFTextureTransformExtension( json ) {
+	function GLTFTextureTransformExtension() {
```
Comment (takahirox) [inline examples/js/loaders/GLTFLoader.js]: No argument need.

File: examples/js/loaders/GLTFLoader.js [hunk 5]
```diff
@@ -646,9 +646,9 @@ THREE.GLTFLoader = ( function () {
 
 			},
 
-			extendParams: function ( params, material, parser ) {
+			extendParams: function ( materialParams, materialDef, parser ) {
```
Comment (takahirox) [inline examples/js/loaders/GLTFLoader.js]: Consistent naming.

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**